src/vivarium_public_health/metrics/treatment.py

```diff
@@ -0,0 +1,73 @@
+from collections import defaultdict
+
+from .utilities import get_age_bins, get_output_template, get_group_counts
+
+
+class TreatmentObserver:
+    """Observes dose counts for a single treatment.
+
+    This component is intended for use with the ``MassTreatmentCampaign``
+    component in ``vivarium_public_health``.  It expects the names of the
+    doses to be supplied under the configuration key
+    ``{treatment_name}.doses`` and should work with any component that
+    specifies its doses in that manner.
+
+    By default, this observer computes aggregate dose counts over the entire
+    simulation.  It can be configured to bin these into age_groups, sexes,
+    and years by setting the ``by_age``, ``by_sex``, and ``by_year``
+    flags, respectively.
+
+    """
+
+    configuration_defaults = {
+        'metrics': {
+            'treatment_observer': {
```

Comment:
I could see myself uses "treatment_observer" directly in the config and getting default behavior accidently. May want to add a note that the config block should be specified with the passed treatment name

Reply:
I agree. Since I'm also confused by this, I tend to add the example to show how config should be given in docstring, not actually for the user, really for reminding myself.